docs/reference/eql/search.asciidoc

```diff
@@ -492,6 +492,7 @@ API returns a response that includes:
 * An `is_partial` value of `true`, indicating the response does not contain
   complete search results.
 * An `is_running` value of `true`, indicating the search is async and ongoing.
+* Partial search results, if available, in the `hits` property.
```

Comment:
We don't have this for now, but hopefully, we will have some partial results soon. So, it might make sense to leave it.

Reply:
I'll leave this in for now. It'll be easy to remove later if wanted.

Reply:
:+1: